docs/modules/ROOT/pages/spring-cloud-bus/bus-endpoints.adoc

```diff
@@ -41,4 +41,32 @@ The `/actuator/busenv` endpoint accepts `POST` requests with the following shape
 	"name": "key1",
 	"value": "value1"
 }
-----
\ No newline at end of file
+----
+
+[[bus-shutdown-endpoint]]
+== Bus Shutdown Endpoint
+The `/actuator/busshutdown` shuts down the application https://docs.spring.io/spring-boot/reference/web/graceful-shutdown.html[gracefully].
+
+To expose the `/actuator/busshutdown` endpoint, you need to add following configuration to your
+application:
+
+[source,properties]
+----
+management.endpoints.web.exposure.include=busshutdown
+----
+
+You can make a request to the `busshutdown` endpoint by issuing a `POST` request.
+
+If you would like to target a specific application you can issue a `POST` request with the bus id:
+
+[source,bash]
+----
+$ curl -X POST http://localhost:8080/actuator/busshutdown/busid
```

Comment:
remove `busid` here?

Reply:
Ah I should change that, its the destination, so it should probably be `{busid}`

Reply:
NVM the example below does that, yes we should remove it.